packages/utils/src/lib/transform.ts

```diff
@@ -113,12 +117,29 @@ export function toUnixNewlines(text: string): string {
   return platform() === 'win32' ? text.replace(/\r\n/g, '\n') : text;
 }
 
+export function fromJsonLines<T = unknown>(text: string) {
+  const unifiedNewLines = toUnixNewlines(text).trim();
+  return JSON.parse(`[${unifiedNewLines.split('\n').join(',')}]`) as T;
+}
+
+export function toJsonLines<T extends object>(objects: T[]) {
```

Comment:
Not all JSON need be an object:

```suggestion
export function toJsonLines<T>(objects: T[]) {
```